### ISSUE TITLE
Feature request: worked example with different imporance density. 

### ISSUE THREAD

**Issue by bsugerman** [jelfring/particle-filter-tutorial#8]
I think there is one additional example that you could add to really complete this tutorial: how would you compute the pdf of the process model $p(x_k|x_{k-1})$. As a hands-on tutorial, that would be of **huge** benefit to people turning the priors, posteriors, and marginalized equations of particle filtering into working code.  Making the assumption that the importance density $q$ is $p(x_k|x_{k-1})$ certainly makes computations of the weight update easier, but also makes it rather opaque to try any other importance density.

**Comment by jelfring** [comment]
I agree about the added value of such an example (it was one of the topic I excluded from the paper due to space limitations). I'd like to add it to the repository some day but unfortunately can't make promises on the timing.
